python/fbprophet/forecaster.py

```diff
@@ -1256,6 +1261,22 @@ def piecewise_logistic(t, cap, deltas, k, m, changepoint_ts):
             m_t[indx] += gammas[s]
         return cap / (1 + np.exp(-k_t * (t - m_t)))
 
+    @staticmethod
+    def flat_trend(t, m):
+        """Evaluate the flat trend function.
+
+        Parameters
+        ----------
+        t: np.array of times on which the function is evaluated.
+        m: Float initial offset.
+
+        Returns
+        -------
+        Vector y(t).
+        """
+        m_t = m * np.ones_like(t)
+        return m_t
+
```

Comment:
@bletham  are you ok with this or would you prefer this just done via `piecewise_linear`  with a keyword which just returns `m` if `flat` ?